### ISSUE TITLE
The components of a hierarchy should not be called `Hierarchy`

### ISSUE THREAD

**Issue by johentsch** [TimeLineAnnotator/desktop#199]
Hierarchy is the name used for the ensemble of components; giving that name to the individual components:

https://github.com/TimeLineAnnotator/desktop/blob/79f4dbf0ece87e2c13e5401fa8c762c486b8b6dd/tilia/timelines/hierarchy/components.py#L22

is terminologically misleading (for example, when I see the button "Delete hierarchy", to me it sounds like it will delete the entire HierarchyTimeline). I would suggest to remain with the graph/tree terminology because these are among the best-known data structures.

**Comment by FelipeDefensor** [comment]
I have to agree that hierarchy is not a good name. As I said in #198, hierarchy timelines are not meant to be strict representations of trees, but I think naming the components "Nodes" would indeed be a great improvement for all users (CS-oriented or not). Thanks for the suggestion!

I guess that also poses the question of whether "hierarchy timeline" should also be changed. "Tree" would not do here, though, for the reason stated above.

**Comment by FelipeDefensor** [comment]
@azfoo anything against renaming "hierarchy" to "node"?

**Comment by azfoo** [comment]
"node" might be quite CS/maths specific way of talking about a graph. "member" would be a little more generic. 
Happy for it to be named "node" anyway.

**Comment by FelipeDefensor** [comment]
> "node" might be quite CS/maths specific way of talking about a graph. "member" would be a little more generic. Happy for it to be named "node" anyway.

How do we feel about "unit", as in "hierarchy unit"?
